.github/workflows/pull-request-new-commit.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: Pull Request New Commit
+
+on:
+  pull_request:
+    types: [synchronize]
+
+jobs:
+  issue_comment_triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          github_token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
does this need permissions? by default the `GITHUB_TOKEN` returned only includes Read rules

Reply:
Not that I can see, I tested this in my own repo and it seems to work. Also used this as a reference and it doesn't seem to need permissions either? https://github.com/hashicorp/terraform-provider-azurerm/blob/main/.github/workflows/pull-request.yaml